community/GOVERNANCE.md

```diff
@@ -9,8 +9,8 @@ The list of current maintainers, their project and organizational affiliation(s)
 
 ## Conflict Resolution and Voting
 The vast majority of issues are resolved by consensus. If consensus cannot be reached, issues are resolved by voting. All decisions should be recorded publicly in GitHub or meeting notes and recordings.
-
 * A vote requires a simple majority to pass. 
+* A vote requires participation by a quorum of 50% of maintainers.
```

Comment:
I am trying to understand the main reason behind this term. I am not challenging it, but to understand the context.

Reply:
Since we have a pretty relaxed maintainer removal process (1yr of inactivity [which I think is a good thing!]), maybe something slightly less than 50%? I just worry about whether we'll be able to get fully 50% of the current list onto a call.

Reply:
Ok, let's start with 30% and tune it from there.